sweep/store.go

```diff
@@ -31,16 +34,91 @@ var (
 	byteOrder = binary.BigEndian
 
 	errNoTxHashesBucket = errors.New("tx hashes bucket does not exist")
+
+	// ErrTxNotFound is returned when querying using a txid that's not
+	// found in our db.
+	ErrTxNotFound = errors.New("tx not found")
 )
 
+// TxRecord specifies a record of a tx that's stored in the database.
+type TxRecord struct {
+	// Txid is the sweeping tx's txid.
+	Txid chainhash.Hash
```

Comment:
Do we also want to store historical information w.r.t the _inputs_ as well? Given that during the course of fee bumping for a given input, it may have been bundled in several distinct transactions before confirmation.